File: Prueba_digital_ware/Prueba_digital_ware/ClientApp/src/app/Models/Venta.ts
import { Cliente } from './Cliente'
import { Empleado } from './Empleado'
import { Producto } from './Producto'

export class Venta {
  VentaId: number
  Fecha: Date
  Cliente: Cliente
  ClienteId: number
  Empleado: Empleado
  EmpleadoId: number
  Total: number
  Productos: Array<Producto>
}

File: Prueba_digital_ware/InventarioData/Model/Venta.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InventarioData.Model
{
    public class Venta
    {
        public int VentaId { get; set; }
        public DateTime Fecha { get; set; }

        public int ClienteId { get; set; }
        public int EmpleadoId { get; set; }

        public decimal Total { get; set; }
    }
}
